### ISSUE TITLE
Cannot identify `keyword colors` in non-pure-color values

### ISSUE THREAD

**Issue by BernieHuang2008** [BernieHuang2008/variablefiy#3]
cannot identify `keyword colors` in non-pure-color values.
e.g.
```css
border: solid 1px black;
```
cannot be detected, but
```css
border: black;
border: solid 1px #000;
```
can both be detected.

**Comment by BernieHuang2008** [comment]
bug near Ln150 of main.html
https://github.com/BernieHuang2008/variablefiy/blob/084e5bfcc10d326e42faa5cbb5e1ccabbff02717/main.html#L150

**Comment by BernieHuang2008** [comment]
solved by pr #4 .
